jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonAssert.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.jdi.light.material.asserts.inputs;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.material.elements.inputs.Button;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class ButtonAssert extends UIAssert<ButtonAssert, Button> {
+
+    @JDIAction("Assert that '{name}' text is '{0}'")
+    public ButtonAssert text(Matcher<String> condition) {
+        jdiAssert(element().getValue(), condition);
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' is enabled")
+    @Override
+    public ButtonAssert enabled() {
+        jdiAssert(element().isEnabled(), Matchers.is(true));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' is disabled")
+    @Override
+    public ButtonAssert disabled() {
+        jdiAssert(element().isDisabled(), Matchers.is(true));
```

Comment:
I guess the behaviour should be the same as for FAB (Mui-disabled)

Reply:
yes, it really should
fixed